client/app/main/student/partial/outreaches.html

```diff
@@ -51,8 +51,13 @@
         </dd>
         <dd ng-repeat="note in outreach.notes | orderBy:$index:true">
           <div class="input-group">
-            <input type="text" class="form-control" readonly value="{{note.note}}"/>
+            <input type="text" class="form-control" readonly value="{{note.note | parseString}}"/>
             <span class="input-group-addon">{{note.date | date:'yyyy-MM-dd'}}</span>
+            <span class="input-group-btn">
+              <button class="btn btn-info" ng-click="viewNote(note, student)">
+                <i class="fa fa-search"></i>
```

Comment:
lets use `<i class="fa fa-ellipsis-h"></i>` rather than search so it doesn't look like a search.